server/api/restClient.ts

```diff
@@ -44,10 +44,6 @@ export default class RestClient {
     return this.config.timeout
   }
 
-  defaultErrorLogger(error: UnsanitisedError): void {
```

Comment:
this was removed to fix a bug inherited from the remplate project - @timharrison-moj has also applied the same fix to CVL